lib/src/oauth2_flows/metadata_server.dart

```diff
@@ -7,6 +7,9 @@ library googleapis_auth.metadata_server_flow;
 import 'dart:async';
 import 'dart:convert';
 
+// Imports the platform-appropriate findMetadataHost().
+import 'metadata_host_stub.dart' if (dart.library.io) 'metadata_host_io.dart';
```

Comment:
This shouldn't be necessary since `lib/src/oauth2_flows/metadata_server.dart` is only referenced from `lib/auth_io.dart` which assumes we running on the VM, where we have  `dart:io`.

Is it referenced from anywhere else?

------
Instead, let's add:
```dart
@TestOn("vm")
```
At the very top of `test/oauth2_flows/metadata_server_test.dart`. This way these tests won't be run in the browser, where it doesn't make sense to run these tests anyways. And the travis job will hopefully pass :)

Reply:
Done.